### DISCUSSION TITLE
v1.0.0-M38

### DISCUSSION THREAD

**Discussion by rossabaker** [http4s/http4s#6903] [Releases]
Tracking releases of the ancillary repos.  Modules where a volunteer maintainer steps up cut to the front of the line! :smile:

* [x] armeria [v1.0.0-M2](https://github.com/http4s/http4s-armeria/releases/tag/v1.0.0-M2)
* [X] blaze [v1.0.0-M38](https://github.com/http4s/blaze/releases/tag/v1.0.0-M38)
* [ ] boopickle
* [ ] dom
* [ ] ~~dropwizard-metrics~~ [maintainer needed](https://github.com/http4s/http4s-dropwizard-metrics/issues/1)
* [ ] fabric
* [ ] feral
* [ ] finagle
* [ ] fs2-data
* [X] jdk-http-client [v1.0.0-M8](https://github.com/http4s/http4s-jdk-http-client/releases/tag/v1.0.0-M8)
* [ ] jetty
* [ ] netty
* [ ] ~~okhttp-client~~ [maintainer needed](https://github.com/http4s/http4s-okhttp-client/issues/6)
* [ ] ~~play-json~~ [maintainer needed](https://github.com/http4s/http4s-play-json/issues/1)
* [X] prometheus-client [v1.0.0-M38](https://github.com/http4s/blaze/releases/tag/v1.0.0-M38)
* [ ] rho
* [X] scala-xml [v1.0.0-M38](https://github.com/http4s/http4s-scala-xml/releases/tag/v1.0.0-M38)
* [X] scalatags [v1.0.0-M38](https://github.com/http4s/http4s-scalatags/releases/tag/v1.0.0-M38)
* [X] servlet [v1.0.0-M38](https://github.com/http4s/http4s-servlet/releases/tag/v1.0.0-M38)
* [ ] session
* [ ] tomcat
* [ ] twirl

**Comment by ybasket** [comment]
The `fs2-data` module joins the milestone train with https://github.com/http4s/http4s-fs2-data/releases/tag/v1.0.0-M39.

**Reply by rossabaker** [reply]
I noted this on the [M39 discussion](https://github.com/http4s/http4s/discussions/6941).  I think you're the first to release explicitly for that.
